Microsoft.Toolkit.Uwp.UI.Controls/StaggeredPanel/StaggeredPanel.cs

```diff
@@ -124,16 +124,9 @@ protected override Size ArrangeOverride(Size finalSize)
                 var child = Children[i];
                 var elementSize = child.DesiredSize;
 
-                double elementWidth = elementSize.Width;
                 double elementHeight = elementSize.Height;
-                if (elementWidth > _columnWidth)
-                {
-                    double differencePercentage = _columnWidth / elementWidth;
-                    elementHeight = elementHeight * differencePercentage;
```

Comment:
We still need to resize the element height if it want's to render larger than we can handle.

Reply:
I don't really understand this. Is there a practical limit on the element's height?

Reply:
@skendrot can you go into more detail here, with the swap of `elementWidth` below it's not used anymore, so is there another piece we're still missing?  Let me know if you think we can get this resolved this week for the 5.1.1 patch or not.

Reply:
Maybe this check never really did anything anyways... If we measure for the column width then the control should fit.

Reply:
It used be that we use `elementWidth` to measure the child, and when the child is wider than the column it scales the bound uniformly.
But now we use `_columnWidth` for measure, so it barely makes any sense. And I think it should be up to the child to do its own resizing, and that might involve needing more height than it originally expected (e. g. wrap the text), so no check is really needed here.
I'll mark this as resolved now.

Reply:
element width should never have been wider than the column because we sized every element to [fit the column width](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Controls/StaggeredPanel/StaggeredPanel.cs#L93).